File: week6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
# conditon for checking inputs
while True:

    Height = get_int("Height: ")

    if Height > 0 and Height < 9:
        break
# printing the values

for i in range(0, Height):
    print((Height-i-1)*" ", end="")
    print("#"*(i+1))

File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("what is the height = ? ");
    }
    while(height < 1 || height > 8);

// atleast it should take once input and condition has to be verified for that we use do while loop always there will be input validation

   for(int i=0; i < height; i++)

   {
       // for loop for printing spaces
       for(int k =0;k <(height -(i+1));k++)
       {
           printf(" ");
       }
       for(int j=0;j<i+1; j++)
       {
           printf("#");
       }
        printf("\n");

   }
}